src/utils/sorting-functions.ts

```diff
@@ -0,0 +1,123 @@
+import { ToDoItem, Priority } from '../models/todoitem-model';
+
+export function sortByTitle(items: ToDoItem[], order: number): ToDoItem[] {
```

Comment:
You can maybe reactor this file and create a function that accepts a property to do the filtering and sorting on, also you do not need two separate functions for the ascending and descending sorting, you can use the `order` variable, you can observe a pattern here.: 
For ascending title you have: 
`
      if (titleA < titleB) {
        return -1;  <- order * -1
      }
      if (titleA > titleB) {
        return 1; <- order
      }
`
For descending you have: 
`
      if (titleA > titleB) {
        return -1; order
      }
      if (titleA < titleB) {
        return 1;  order * -1
      }
`
You just need to reorder the conditions for descending for example, and you will see that they are the same code basically: 
`
      if (titleA < titleB) {
        return 1;  <- order * -1
      }
      if (titleA > titleB) {
        return -1; <- order
      }
`

Please try to apply the same for every property.

And if you get that try and look for similarities again, this may be done in a single function, if you provide the filtering property, with dynamic object property access.